### PR TITLE
feat(Blinest): add presence

### PR DESCRIPTION
**Pull request by mchev**
## Description 
Add presence for Blinest

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

Screenshot1

Screenshot2
</details>

### CODE REVIEW

File: websites/B/Blinest/metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "828916578698526722",
+		"name": "blinest#3458"
```
Comment (theusaf) [inline websites/B/Blinest/metadata.json:5]: Discriminator added by mistake in old `pmd`
```suggestion
		"name": "blinest"
```

File: websites/B/Blinest/metadata.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	},
+	"service": "Blinest",
+	"description": {
+		"en": "Blinest is a free multiplayer music quiz site."
+	},
+	"url": "blinest.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/tAwvgnB.png",
```
Comment (theusaf) [inline websites/B/Blinest/metadata.json:13]: Logos should be a **square** with dimensions of [at least 512x512px](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo).

File: websites/B/Blinest/presence.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+presence.on("UpdateData", async () => {
+	presence.setActivity({
+		details: "Blinest",
+		state: "Playing on Blinest",
+		buttons: [
+			{
+				label: "Play",
+				url: "https://blinest.com/",
+			},
+		],
```
Comment (theusaf) [inline websites/B/Blinest/presence.ts:14]: Buttons that lead to home pages are [not allowed](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets)
```suggestion
```

**Review by theusaf — CHANGES_REQUESTED** [review]
Low quality. Add more paths/details.

**Comment by theusaf** [comment]
Screenshots also do not depict the presence working.

**Comment by EncryptedDev** [comment]
Low quality

**Pull request closed without merging**